bittensor/__init__.py

```diff
@@ -286,6 +286,9 @@ def debug(on: bool = True):
     ProposalCallData,
     ProposalVoteData,
 )
+
+from . import subtensor as subtensor_module
+from bittensor import subtensor as subtensor_module
```

Comment:
I think we accidentally added double imports here

Reply:
Great catch! Done